src/vmprof_main.h

```diff
@@ -175,11 +179,43 @@ static PY_THREAD_STATE_T * _get_pystate_for_this_thread(void) {
 }
 #endif
 
+#ifdef VMPROF_LINUX
+static void broadcast_signal_for_threads(pid_t pid)
```

Comment:
Ah, good call. In turn:
1. What can we do if they're *not* signal safe?
2. I've actually tried to find another API for thread enumeration and failed, I don't like the readdir solution, either. (Although technically it's asking the kernel, not the disk.)